src/components/FeaturedCard/FeaturedCard.tsx

```diff
@@ -33,34 +43,39 @@ export class FeaturedCard extends Component<Props> {
   }
 
   render() {
+    const {API, Auth, Cors, HTTPS, Category, Description, Link, favorites} = this.props
+    const { note, ...originalApiData} = this.state
     return (
       <div className="featured-card">
-        <h2>{this.props.API}</h2>
-        <p>Authentication: {this.props.Auth}</p>
-        <p>Cors? {this.props.Cors}</p>
-        <p>{this.props.HTTPS}</p>
-        <p>Category: {this.props.Category}</p>
-        <p>{this.props.Description}</p>
-        <p>
-          <a href={this.props.Link} target="_blank">
-            more details
+        <HomeLink to='/' className='home-link'>
+          <BiArrowBack className="home-link__arrow" />View all apis
+        </HomeLink>
+        <div className="favorite-btn-container">
```

Comment:
Good call adding this return button on the Feature Card - improves navigability a lot. One thing we'll need to fix: If you conduct any kind of search, click on an API card and then click the 'view all' button, the search form on the homescreen is reset but only the search results are still visible.